src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -96,6 +96,26 @@ def filter_snuba_results(snuba_results, groups_to_backfill_with_no_embedding, pr
     return filtered_snuba_results, groups_to_backfill_with_no_embedding_has_snuba_row
 
 
+def create_project_cohort(worker_number: int, last_processed_project_id: int | None) -> list[int]:
+    """
+    Create project cohort by the following calculation: project_id % threads == worker_number
+    to assign projects uniquely to available threads
+    """
+    project_id_filter = Q()
+    if last_processed_project_id is not None:
+        project_id_filter = Q(id__gt=last_processed_project_id)
+    thread_number = options.get("similarity.backfill_seer_threads")
```

Comment:
this option already existed and represents the number of concurrent requests we'd make to seer, but this is something different no? shouldn't thread number be passed into the backfill?

Reply:
right, but `thread_number` is already used and means something different, no?

Reply:
ah I thought they were the same thing, I'll make it an option instead of another parameter to pass in, as it should be constant